File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ShadowPotion.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class ShadowPotion extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_shadow_potion_name;
        this.idDescription = R.string.item_shadow_potion_description;
        this.idImage = R.drawable.shadow_potion;
        this.price = 591L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/ShadowPotion.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class ShadowPotion : Item() {
    override fun configureProperties() {
        idName = R.string.item_shadow_potion_name
        idDescription = R.string.item_shadow_potion_description
        idImage = R.drawable.shadow_potion
        price = 591L
    }
}
